client/src/components/Header/Header.component.jsx

```diff
@@ -189,7 +189,7 @@ const Header = () => {
             maxW="680px"
             w="100%"
           >
-            <SearchBoxComponent agencyId={agency?.id} />
+            <SearchBox agencyId={agency?.id} />
```

Comment:
Not this line, but for line # 290 of the same file, you forgot to change `SearchBoxComponent` to `SearchBox`

Reply:
lgtm otherwise, but didn't want to approve in case this isn't addressed (it will break the app)